scripts/konflux_resolve.py

```
@@ -0,0 +1,1179 @@
+        collector = _LinkCollector()
+        collector.feed(html)
+        return collector.link_texts
+
+    @staticmethod
+    def parse_package_page(html: str) -> list[dict[str, Any]]:
+        """Return list of entry dicts from a per-package page."""
+        collector = _LinkCollector()
+        collector.feed(html)
+        entries: list[dict[str, Any]] = []
+
+        for href, link_text in zip(collector.hrefs, collector.link_texts):
```

Ruff (0.15.21)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)